databricks/koalas/dask/utils.py

```diff
@@ -1,56 +1,6 @@
 import functools
 from databricks.koalas.dask.compatibility import get_named_args
 
-_method_cache = {}
-
-
-class methodcaller(object):
```

Comment:
I think these are not used. mypy complained so I just removed.

Reply:
That sounds great, thanks for the cleanup.